src/sql/src/parsers/tql_parser.rs

```diff
@@ -26,12 +26,14 @@ pub const TQL: &str = "TQL";
 const EVAL: &str = "EVAL";
 const EVALUATE: &str = "EVALUATE";
 const EXPLAIN: &str = "EXPLAIN";
+const VERBOSE: &str = "VERBOSE";
+
 use sqlparser::parser::Parser;
 
 /// TQL extension parser, including:
 /// - `TQL EVAL <query>`
-/// - `TQL EXPLAIN <query>`
-/// - `TQL ANALYZE <query>`
+/// - `TQL EXPLAIN (VERBOSE) <query>`
+/// - `TQL ANALYZE (VERBOSE) <query>`
```

Comment:
```suggestion
/// - `TQL EXPLAIN [VERBOSE] <query>`
/// - `TQL ANALYZE [VERBOSE] <query>`
```

To indicate that an item is optional, prefer to use the bracket notation `[item]`.